sale_partner_source/hooks.py

```diff
@@ -0,0 +1,26 @@
+import logging
+
+_logger = logging.getLogger(__name__)
+
+
+def _post_init_sale_partner_source(env):
+    cr = env.cr
+    # Create source from partner category
+    cr.execute("""
+        INSERT INTO utm_source (name, create_uid, create_date, write_uid, write_date)
+        SELECT name, create_uid, create_date, write_uid, write_date
+        FROM partner_classification
+    """)
```

Comment:
field name in **utm.source** model is a unique constraint field. Have you considered the case of inserting duplicate name?

Reply:
One more thing, we will migrate data from **`partner.category`** model and **`gts_partner_category`** module. Not from **`partner.classification`**